### ISSUE TITLE
How do I find out what rule was triggered?

### ISSUE THREAD

**Issue by alexolog** [sixcious/infy-scroll#74]
### Feature Request

I have set a custom rule for a website.
Some pages on that site paginate (incorrectly!) even though the `nextLink` that I set is not found.
Apparently some other other rule is triggered, probably from the DB.
How can I find what rule triggered the pagination, so I can disable it?

### Adherence

- [X] My feature request is an actual feature and is not related to making it work with a specific website

**Comment by sixcious** [comment]
Hi Alex!

I added a new section in the Help Guide that covers this in more detail here:
https://github.com/sixcious/infy-scroll/wiki/Activation-and-Prioritization#activation-indication

Basically, when you click on the toolbar icon, the UI Window will open, and if Infy has already activated itself, it will open in a minimal view as shown in the link above. In the top-right corner, there will be an icon that indicates how it activated. If you see a heart icon, it's from one of your saves. If you see a database icon, it's from one of the databases. You can hover over the icon for more details about the URL pattern it's using if needed.

Hope this helps 💜

**Comment by alexolog** [comment]
Thanks!

The solution was to disable the DB on that URL (the documentation says that blacklisting a URL will stop Infy from working on the URL, which is not entirely correct since the saves still work)

Is it possible to only blacklist some of the DBs on a URL/pattern?

**Comment by sixcious** [comment]
Hi again!

> the documentation says that blacklisting a URL will stop Infy from working on the URL, which is not entirely correct since the saves still work

Hmm... well, the text input for the blacklist is labeled `Database Blacklist` and it is only mentioned and found in the Databases section in the Options and Wiki. You can let me know the specific sentence(s) you found that implies it also applies to Saves and I can reword it to be more clear that it only applies to the Database.

> Is it possible to only blacklist _some of the DBs_ on a URL/pattern?

Sorry, I'm not sure I understand the question. There's only _two databases_ and the InfyScroll Database doesn't really contain the same URLs/items as the AutoPagerize Database.

**Comment by alexolog** [comment]
There's an option to add custom databases.

**Comment by sixcious** [comment]
Hi again, there is no option to add a custom database in the current version of Infy Scroll (Version 0.8). This is explained in the [FAQ](https://github.com/sixcious/infy-scroll#what-happend-to-the-custom-database-and-save-whitelist). Please note that Saves and Databases are completely separate collections and are going to have separate settings. I'm going to close this as I answered the original question.

Thank you.

**Comment by alexolog** [comment]
Thank you for the answer.
Can you remove the "custom database" button though.

**Comment by sixcious** [comment]
Hi, that is a button link to the FAQ that explains what happened to the Custom Database that used to be in Version 0.7. If I didn't put that button there, then users would have had no idea what happened to it in Version 0.8. I plan on removing it in the next version, but not putting it there for the last update would have been a huge problem.

Thank you.
